templates/patient/demographics/relation_form_search.html.twig

```diff
@@ -287,16 +287,36 @@
 <script type="text/javascript">
     // ============= UNIFIED PERSON SEARCH AND CREATE =============
 
-    let csrfToken = document.getElementById('addPersonSection')?.dataset.csrfToken;
+    const tableId = {{ table_id|js_escape|raw }};
+    const webroot = {{ webroot|js_escape|raw }};
```

Comment:
remove `|raw` in above 2 lines since does nothing (we do not use auto-escaping in twig, so no need for this)